schemas/asset-schema.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "asset-schema.json",
+  "title": "AssetEvent",
+  "type": "object",
+  "description": "Schema of ",
+  "required": ["type", "application"],
+  "properties": {
+    "type": {
+      "const": "resource"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Are we planning to have other types? If so, what would it look like? I'm asking, so we can be sure that the `asset` naming and this `type:resource` property make sense

Reply:
The other type is segment, as far as I know we do not plan to have other type